calico_cni/calico_cni.py

```diff
@@ -152,12 +152,13 @@ def execute(self):
         except SystemExit, e:
             # SystemExit indicates an error that was handled earlier
             # in the stack.  Just set the return code.
-            rc = e.code 
+            rc = e.code
         except BaseException:
             # An unexpected Exception has bubbled up - catch it and
             # log it out.
             _log.exception("Unhandled Exception killed plugin")
-            rc = 1
+            rc = ERR_CODE_GENERIC
```

Comment:
This guy can still be ERR_CODE_UNHANDLED.  This is the only place that should be used, because it is hit when an unhandled exception is thrown :)